### PR TITLE
1526 poc ldraney

### PR DESCRIPTION
**Pull request by cris-oddball**
easier to add comments if there is a PR.

### CODE REVIEW

File: .github/actions/workflow-trigger-and-wait/action.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    required: false
+  wait_interval:
+    description: 'Time interval in seconds between status checks'
+    required: false
+    default: '10'
+runs:
+  using: 'composite'
+  steps:
+    - run: bash ${{ github.action_path }}/trigger_script.sh "${{ inputs.owner }}" "${{ inputs.repo }}" "${{ inputs.github_token }}" "${{ inputs.workflow_file_name }}" "${{ inputs.ref }}" "${{ inputs.client_payload }}" "${{ inputs.wait_interval }}"
+      shell: bash
```
Comment (cris-oddball) [inline .github/actions/workflow-trigger-and-wait/action.yml:30]: probably needs the outputs section
```
outputs:
  workflow_id:
    description: The ID of the workflow that was triggered by this action
  workflow_url:
    description: The URL of the workflow that was triggered by this action
  conclusion:
    description: Conclusion of the job, i.e pass/failure
```

File: .github/actions/workflow-trigger-and-wait/trigger_script.sh [hunk 2]
```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+set -e
+
+# Arguments
+INPUT_OWNER="${1}"
+INPUT_REPO="${2}"
+INPUT_GITHUB_TOKEN="${3}"
+WORKFLOW_FILE_NAME="${4}"  # This should be the workflow file name or ID
+REF="${5}"
```
Comment (cris-oddball) [inline .github/actions/workflow-trigger-and-wait/trigger_script.sh:9]: note that this means REF is required in the workflow

File: .github/actions/workflow-trigger-and-wait/trigger_script.sh [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+CLIENT_PAYLOAD="${6}"  # JSON payload for inputs
+
+# GitHub API URL
+GITHUB_API_URL="https://api.github.com"
+
+# Function to make API calls
```
Comment (cris-oddball) [inline .github/actions/workflow-trigger-and-wait/trigger_script.sh:15]: probably too much was stripped from this file (all the validations)

**Comment by ldraney** [comment]
closing this to remove from my PR feed :)

**Pull request closed without merging**